optaplanner-docs/src/main/asciidoc/RepeatedPlanning/RepeatedPlanning-chapter.adoc

```diff
@@ -488,3 +488,24 @@ A `BestSolutionChangedEvent` does not guarantee that every `ProblemFactChange` h
 ----
 
 . Use `Score.isSolutionInitialized()` instead of `Score.isFeasible()` to only ignore uninitialized solutions, but do accept infeasible solutions too.
+
+[[multiStagePlanning]]
+== Multi-stage Planning
+
+In multi-stage planning, complex planning problems are broken down in multiple stages.
+A typical example is train scheduling, where one department decides where and when a train will arrive or depart,
+and another department assigns the operators to the actual train cars/locomotives.
+
+Each stage has its own solver configuration (and therefore its own ``SolverFactory``):
+
+image::RepeatedPlanning/multiStagePlanning.png[align="center"]
+
+Planning problems with different publication deadlines must use multi-stage planning.
+But also problems with the same publication deadline, solved by different organizational groups
```

Comment:
But problems with the same publication deadline, solved by different organizational groups
are also initially better off with multi-stage planning, because to Conway's law
and the high risk associated with unifying such groups.

IBM SG:
because of prep
Use “because of,” not “due to,” in adverbial clauses.
Although the use of “due to” as a synonym of “because of” has some acceptance in modern English usage, it
is imprecise and, therefore, not suited to technical writing. Incorrect: “Due to the power failure, the update
stopped.” Correct: “Because of the power failure, the update stopped.”